.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["Chocolatecake"]
```

Comment:
I probably wouldn't commit VS Code settings like this. If you don't want them to be tracked by git (because they're your local settings rather than something repo-specific) then you can use a `.gitignore` file: https://www.w3schools.com/git/git_ignore.asp